### PR TITLE
New: Simple template titles registration improvement

### PR DESCRIPTION
**Pull request by alexmigf**
This PR enhances the registration of Simple template titles by relocating them to the document abstract, rather than injecting them directly into the template. This approach is particularly beneficial when using multilingual plugins, especially TranslatePress.

### CODE REVIEW

**Review by MohamadNateqi — CHANGES_REQUESTED** [review]

File: includes/wcpdf-functions.php [hunk 1]
```diff
@@ -959,3 +959,23 @@ function_exists( 'has_block' ) &&
...
 	return $is_block;
 }
+
+/**
+ * Get the default table headers for the Simple template.
+ *
+ * @param object $document
+ * @return array
+ */
+function wpo_wcpdf_get_simple_template_default_table_headers( $document ): array {
```
[the remark this answers is not in the document]
Comment (alexmigf) [reply includes/wcpdf-functions.php:969]: This is only used in the Simple Template, the other templates use `wpo_wcpdf_templates_get_table_headers()`. I don't have a preference, but what you think?

File: includes/documents/class-wcpdf-packing-slip.php [hunk 2]
```diff
@@ -12,6 +12,11 @@
  */
 
 class Packing_Slip extends Order_Document_Methods {
+	
+	public $type;
+	public $title;
+	public $icon;
+	public $output_formats;
```
[the remark this answers is not in the document]
Comment (alexmigf) [reply includes/documents/class-wcpdf-packing-slip.php:19]: What do you mean?

File: templates/Simple/invoice.php [hunk 3]
```diff
@@ -103,23 +102,26 @@
...
 <table class="order-details">
+	<?php $headers = wpo_wcpdf_get_simple_template_default_table_headers( $this ); ?>
 	<thead>
 		<tr>
-			<th class="product"><?php _e( 'Product', 'woocommerce-pdf-invoices-packing-slips' ); ?></th>
-			<th class="quantity"><?php _e( 'Quantity', 'woocommerce-pdf-invoices-packing-slips' ); ?></th>
-			<th class="price"><?php _e( 'Price', 'woocommerce-pdf-invoices-packing-slips' ); ?></th>
+			<?php 
+				foreach ( $headers as $column_class => $column_title ) {
+					printf( '<th class="%s"><span>%s</span></th>', $column_class, $column_title );
```
[the remark this answers is not in the document]
Comment (alexmigf) [reply templates/Simple/invoice.php:110]: I'm just following the same pattern as the Premium Templates, but I will remove.

File: includes/wcpdf-functions.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (MohamadNateqi) [reply includes/wcpdf-functions.php:969]: My idea was to make it a general function so customers can also use it for their custom templates. I'm OK with this too.

File: includes/documents/class-wcpdf-packing-slip.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (MohamadNateqi) [reply includes/documents/class-wcpdf-packing-slip.php:19]: These properties already declared in the parent class, so I think maybe we can not declare them again here?

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Pull request merged**